components/collections/Add.vue

```diff
@@ -58,19 +59,22 @@ export default {
     syncCollections() {
       if (fb.currentUser !== null && this.SYNC_COLLECTIONS) {
         fb.writeCollections(
-          JSON.parse(JSON.stringify(this.$store.state.postwoman.collections)),
-          "collections"
+          this.$props.type == "rest"
+            ? JSON.parse(JSON.stringify(this.$store.state.postwoman.collections))
+            : JSON.parse(JSON.stringify(this.$store.state.postwoman.collectionsGraphql)),
+          this.$props.type == "rest" ? "collections" : "collectionsGraphql"
```

Comment:
this is an overkill. can't we use something like this..?

```suggestion
          JSON.parse(JSON.stringify(this.$props.type == "rest" ? this.$store.state.postwoman.collections : this.$store.state.postwoman.collectionsGraphql), this.$props.type == "rest" ? "collections" : "collectionsGraphql")
```